src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -557,10 +558,12 @@ export default {
 						withRemoving: event.data.withRemoving,
 					})
 					break
-				case 'update-nextcloud-talk-hash':
-					this.$store.dispatch('setNextcloudTalkHash', event.data.hash)
+				case 'update-nextcloud-talk-hash': {
+					const talkHashStore = useTalkHashStore()
+					talkHashStore.setNextcloudTalkHash(event.data.hash)
 					break
 				}
+				}
```

Comment:
Linter was complaining about declaring a const inside of case block

UPD: Replaced with following, still works:
```js
	case 'update-nextcloud-talk-hash':
		useTalkHashStore().setNextcloudTalkHash(event.data.hash)
		break
```

Reply:
`composables`, including Pinia stores, should be used only during instance composition — in the `setup()` function. Otherwise, it doesn't have instance context, because there is no current setting up instance. 

It works with Pinia, because it has active pinia cached in a global variable, but it may cause unexpected issues in different cases.

For example, if there is more than 1 pinia in the application (e.g. Talk Desktop will have its own Pinia), or if we use provide/inject in store.